### ISSUE TITLE
ImageBuildDetails.wait_for_completion(show_output=True) raises AttributeError: 'NoneType' object has no attribute 'splitlines'

### ISSUE THREAD

**Issue by ezwiefel** [Azure/MachineLearningNotebooks#710]
Code Example:
```python
env = ws.environments['foo']

image_build = env.build(ws)
image_build.wait_for_completion(show_output=True)
```

Results:
```
Image Build Status: Running

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-12-df9e24646f34> in <module>
----> 1 image_build.wait_for_completion(show_output=True)

/anaconda/envs/azureml_py36/lib/python3.6/site-packages/azureml/core/environment.py in wait_for_completion(self, show_output)
    373                 if show_output:
    374                     content = self.get_log()
--> 375                     printed = _incremental_print(content, printed, file_handle)
    376                 else:
    377                     print('.', end='', flush=True)

/anaconda/envs/azureml_py36/lib/python3.6/site-packages/azureml/core/environment.py in _incremental_print(log, printed, fileout)
    339         def _incremental_print(log, printed, fileout):
    340             count = 0
--> 341             for line in log.splitlines():
    342                 if count >= printed:
    343                     print(line, file=fileout)

AttributeError: 'NoneType' object has no attribute 'splitlines'
```

**Comment by YutongTie-MSFT** [comment]
@ezwiefel Hi Erik, could you please provide more information about your issue?

**Comment by ezwiefel** [comment]
What other information can I provide? When I tried to build an environment and then wait for completion, I got the error above.

**Comment by ezwiefel** [comment]
@YutongTie-MSFT I just tried running the code below - you should be able to replicate with this code in a Notebook VM.
```python
import azureml.core
from azureml.core import Workspace, Environment

print(azureml.core.VERSION)

ws = Workspace.from_config()
env = Environment.get(ws, 'AzureML-Minimal')

build = env.build(ws)
build.wait_for_completion(show_output=True)
```

And got this result:
![image](https://user-images.githubusercontent.com/8908818/71681779-3bfa0d00-2d53-11ea-9b3d-901d2454d22f.png)

**Comment by ezwiefel** [comment]
@YutongTie-MSFT - I figured out what the issue is here. The `Environment.get_log` method is missing the return command. It should be:

```python
def get_log(self):
    """Return image build log."""
    return requests.get(self.log_url, stream=True).text
```

**Comment by vizhur** [comment]
Thank you @ezwiefel  your fix will be shipped with upcoming release!

**Comment by vizhur** [comment]
The fix is released with 1.0.85 version
